src/main/java/me/totalfreedom/totalfreedommod/AutoEject.java

```diff
@@ -85,6 +85,10 @@ else if (kicks >= 3)
                 player.kickPlayer(kickMessage);
                 break;
             }
+            default:
```

Comment:
Move this to last

Reply:
but then it wouldn't default to STRIKE_THREE

Reply:
Have you tested this?

Reply:
don't need to, the default thing doesn't have a break in it so it ends up going to STRIKE_THREE after sending the message about the ejectmethod not being recognized into the logs. if we moved it to last, it would give the warning but nothing else would happen

Reply:
I didn't see the `default` body so please ignore my comment.